README.md

```diff
@@ -1 +1,42 @@
-# restaurant-page
\ No newline at end of file
+# Restaurant page
+
+> This project consisted on building a simple restaurant page for practicing tabbed browsing.
+
+![screenshoot](./src/assets/restaurant-page.png)
+
```

Comment:
Let's update the README file with `Link Link` to complete the 3 basic rules to follow in Microverse projects.

https://github.com/microverseinc/readme-template

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

Reply:
@AlduLonghi, You have really done well working on this project, but for some reason, you didn't add `live link` as required in the previous review.

So think about it this way;
1. You should know that all Microverse students follow the 3 basic rules stated in my comment above.
2. This is your repo, employer will definitely come around checking out your project, they will want to check your design/work within a minute.
3. You might want to add this to your portfolio, I bet you don't want people to visit your portfolio and they would have to `run build` before they could view your design.

kindly add the live link.

Reply:
@bellom Oh! I am so sorry. I did not understand well what you were referring to since in the first review it says "Link link". Now I get it.

Reply:
@AlduLonghi, so sorry, I guess I should have written `live demo link` just as written in the template.